commons/src/main/java/org/eclipse/kapua/commons/configuration/AbstractKapuaConfigurableService.java

```diff
@@ -129,7 +129,7 @@ private void validateConfigurations(String pid, KapuaTocd ocd, Map<String, Objec
                 }
             }
 
-            String logicValidationResult = validateNewConfigValuesCoherence(ocd, updatedProps, scopeId);
+            String logicValidationResult = validateNewConfigValuesCoherence(ocd, updatedProps, scopeId, parentId);
```

Comment:
It seems that here the validateNewConfigValuesCoherence at line 151 if called, and it always returns `""` , so no validation is performed on changing configuration values.

I would move this after the current line 138 so we check values after full configuration validation.